promptsource/templates/jigsaw_unintended_bias/templates.yaml

```diff
@@ -0,0 +1,212 @@
+dataset: jigsaw_unintended_bias
+templates:
+  17f730bd-fc7e-419c-84cc-e5c795268d02: !Template
+    answer_choices: null
+    answer_choices_key: null
+    id: 17f730bd-fc7e-419c-84cc-e5c795268d02
+    jinja: 'Online comments can sometimes be toxic. For instance, they might be insulting, threatening, racist or
+    many other issues. If 0 corresponds to a very civil comment, and 100 to a highly toxic one, how would you grade the
+    following comment?
+
+    "{{comment_text}}"
+
+    |||
+
+    {{(target * 100.0) | round(3) }}
+    '
+    metadata: !TemplateMetadata
+      choices_in_prompt: null
+      metrics:
+      - Other
+      original_task: true
+    name: online_comment
+    reference: ''
+  1add6d25-346e-4bf5-aa58-8dc184018f9e: !Template
+    answer_choices:
+    - keep
+    - remove
+    answer_choices_key: null
+    id: 1add6d25-346e-4bf5-aa58-8dc184018f9e
+    jinja: 'You are the moderator of the comments section on a website. You want to remove uncivil comments.
+    Would you {{answer_choices[1]}} or {{answer_choices[0]}} the following comment?
+
+    "{{comment_text}}"
+
+    |||
+
+    {{answer_choices[target | round | int]}}
+    '
+    metadata: !TemplateMetadata
+      choices_in_prompt: true
+      metrics:
+      - Other
+      original_task: true
+    name: moderator
+    reference: ''
+  355aa574-61c2-4b94-9f1a-9347fd7c6444: !Template
+    answer_choices: null
+    answer_choices_key: null
+    id: 355aa574-61c2-4b94-9f1a-9347fd7c6444
+    jinja: "{{toxicity_annotator_count}} annotators where tasked with assessing the\
+      \ toxicity of the following comment:\n\n\"{{comment_text}}\"\n\nOn a scale of\
+      \ 0 to 1, what do you think their average prediction was?\n\n||| \n\n{{ target\
+      \ | round(5) }}"
```

Comment:
weak convention, let's round it to one digit after comma as on in https://github.com/bigscience-workshop/promptsource/blob/d8ea175abdb5aab733c28a5b90fd39762764dc9e/promptsource/templates/glue/stsb/templates.yaml#L114
it seems like it worked well for t5

Reply:
You're right, this is probably more precision than is needed. I kept it to one digit when the score is normalized to be between [0,  1] or [0, 5] and to 0 digits when it is in [0, 100].

Note that I chose different scales for the diversity of prompts / better generalization if we trained on them. Since we might be using those as eval only, this might not be what we want. I would hope the model is resilient to those but it might make any postprocessing script more tricky than warranted.

Reply:
thanks!